export.py

```diff
@@ -2418,7 +2418,7 @@ def export_render_settings(ri, rpass, scene, preview=False):
     ri.Attribute("trace", depths)
     if rm.use_statistics:
         ri.Option("statistics", {'int endofframe': 1,
-                                 'string xmlfilename': 'stats.xml'})
+                                 'string xmlfilename': 'stats_%d.xml' % scene.frame_current})
```

Comment:
To make this more consistent with the rib file maybe do %04d ?  stats_0001.xml vs stats_1.xml

Also I'd prefer stats.0001.xml